content/en/blog/2023/announcing-notation-rc2.md

```diff
@@ -0,0 +1,96 @@
+
+```
+$ notation verify --user-metadata io.wabbit-networks.buildId=123 sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Successfully verified signature for sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+The artifact is signed with the following user metadata.
+
+KEY                         VALUE
+io.wabbit-networks.buildId  123
+```
+
+In addition, storing signatures in the registry using OCI image manifest was added into RC.3, you can use a flag `--signature-format image` to explicitly specify uploading image manifest to a registry when signing an artifact.
+
+Last but not least, `--debug` and `--verbose` flags were added to all CLI commands, providing debug and troubleshooting capability for Notation CLI.
+
+### Ecosystem integration
+
+[Ratify](https://github.com/deislabs/ratify) is a verification engine for Kubernetes which enables verification of artifact security metadata and admits for deployment only those that comply with policies you create. It integrated Notation-go v1.0.0-rc.2 which allows you to verify a container image signed by Notation.
+
+[Zot registry](http://zotregistry.io/) also integrates notation-go and supports storing signatures associated with the artifact. See this [demo](https://github.com/project-zot/zot/tree/main/demos#notation) for details.
+
+Going forward, Notation maintainers will work the community to support more ecosystem tooling and deliver consistent signing and verification experience, such as HashiCorp Vault KMS plugin, support Kyverno for Kubernetes policy management, and being compatible with more OCI registries.
+
+### Enhanced stability with more testing coverage
+
+As the second RC release for v1.0.0, we aim to deliver a production-ready security product.  Comparing with RC.1, the E2E testing framework built on [ginkgo](https://onsi.github.io/ginkgo/) was set up in RC.2 and major test cases were added to Notation CLI. Meanwhile, unit test coverage was also increased by [5.92%](https://app.codecov.io/gh/notaryproject/notation-go?search=&trend=3%20months). It definitely enhanced the program robustness and project stability.
+
+## Credits
+
+We would like to specially thank the Notation maintainers, contributors, and the broader Notation community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release. Sending credits to the following contributors who made great contributions to RC.2.
+
+@toddysm
```

Comment:
I would put those in alphabetical order. I am not sure my name should be first in that list from contribution point of view ;) and this gives wrong impression. Alphabetical order is at least impartial.

Reply:
Good catch, I will adjust this contributor list in alphabetical order

Reply:
Updated to use alphabetical order